content/md/en/docs/tutorials/get-started/build-local-blockchain.md

```diff
@@ -69,8 +69,8 @@ To compile the Substrate node template:
    git clone https://github.com/substrate-developer-hub/substrate-node-template
    ```
 
-   In most cases, you can clone the `main` branch to get the latest code.
-   However, you can use the `--branch` command-line option if you want to work with a Substrate branch that is compatible with a specific Polkadot version.
+   This will put you into the `main` branch. If you are experienced with Polkadot, and want to work with a specific Polkadot version, you can use the `--branch` command-line option to select the Substrate branch that you want to work with.
```

Comment:
```suggestion
   This command clones the `main` branch. 
   If you are familiar with Polkadot, and want to work with a specific Polkadot version, you can use the `--branch` command-line option to select the specific branch that you want to work with.
```
Typically, we want to avoid indefinite articles and future tense. If you specify a branch in this repo it is really a **node template** branch that is compatible with a specific **Polkadot** version. (The tags are all polkadot releases.)

Reply:
OK.  My goal is to just change the language to be less....optional.  Especially for a beginner tutorial, every time a person has to make a decision about things that they may not fully understand the ramifications for, it is a place where people will stumble, and we will lose some of them. I like your changes to my wording. Thank you.